src/processors/downgrade-unmodified-lines.js

```diff
@@ -0,0 +1,15 @@
+const fs = require( 'fs' );
+const downgradeUnmodifiedLines = require( '../lib/downgrade-unmodified-lines' );
+const differ = require( '../lib/differ' );
+const gitDiffCalculator = require( '../lib/git-diff-calculator' );
+
+const config = JSON.parse( fs.readFileSync( '.eslines.json', 'utf-8' ) );
+
+module.exports = function( report ) {
+	const remote = config.processors[ 'downgrade-unmodified-lines' ].remote || 'origin/master';
```

Comment:
Good catch, will prepare a fix for that in other PR. 

Regarding the formatter API, I'd rather use the public API for [ESLint formatters](https://eslint.org/docs/user-guide/formatters/), so we can still do `eslint -f ./path/downgrade-unmodified-lines`. I reckon it is not a common case, but will help in integrating `eslines` into other scenarios (such as integrating it into editors, which I'm hoping we can have the energy to address some day).